### ISSUE TITLE
one query 

### ISSUE THREAD

**Issue by nehaleosharma** [junyanz/pytorch-CycleGAN-and-pix2pix#741]
Previously you had resolved all the queries and appreciated for the frequent reply .With the ray of hope i need help in another problem too
https://github.com/eriklindernoren/PyTorch-GAN(im using cGAN from here)it is completing 200 epochs on anaconda prompt but im not able to get the results and want to use RGB images .
if possible kindly guide me.

Thanks  a lot
![200 epochs complete](https://user-images.githubusercontent.com/51158296/63589207-25ce8a00-c5c6-11e9-99bf-4b980ea2d4c5.png)

**Comment by junyanz** [comment]
Did you post the question in the wrong repo?

**Comment by nehaleosharma** [comment]
no, i just need a help and i couldn't think of anything better.
